alchemy/src/vercel/project-domain.ts

```diff
@@ -157,16 +154,17 @@ export const ProjectDomain = Resource(
       }
 
       case "delete": {
-        const api = await createVercelApi({
-          baseUrl: "https://api.vercel.com/v9",
-          accessToken,
-        });
         await api.delete(
           `/projects/${props.project}/domains/${this.output.name}`,
         );
 
         return this.destroy();
       }
+
+      // todo: make 'dev' phase backwards compatible
+      default: {
+        throw new Error(`Invalid phase: ${this.phase}`);
+      }
```

Comment:
Yeah a qualifier will help. Create, update and delete are all valid during dev mode. It's a different property on context